cql3/query_processor.cc

```diff
@@ -1025,16 +1026,30 @@ query_processor::execute_schema_statement(const statements::schema_altering_stat
 
     cql3::cql_warnings_vec warnings;
 
+    auto request_id = guard->new_group0_state_id();
+    // TODO: either save as a member of statements::schema_altering_statement class
+    //       or extend alter_keyspace_statement::prepare_schema_mutations with guard's req id
```

Comment:
isn't the code below implementing the first TODO option already?

Reply:
Yes, the 1st version is implemented, perhaps the comment could be refined, I'll fix it.
Out of these proposed 2 versions, the 1st one doesn't impact the code that much, but is a bit hacky, IMO the proper way to do it would be to follow the 2nd option. I left this TODO to decide which version to stick to.

Reply:
This question relates to https://github.com/scylladb/scylladb/pull/16723#discussion_r1587115923, if it's not the same, so I'll close this conversation and we can continue to talk about it in this other thread.